backend/functions/package.json

```diff
@@ -6,7 +6,7 @@
     "format": "prettier --write src/**/*.ts",
     "format:check": "prettier --check src/**/*.ts",
     "build": "tsc",
-    "serve": "concurrently \"yarn build -w\" \"firebase emulators:start --only functions\"",
```

Comment:
We need `concurrently` on the backend for this command to work nicely on Windows with hot reloading.

Reply:
Updated